### PR TITLE
Show `gp idp` in the CLI help menu

### PR DESCRIPTION
**Pull request by filiptronicek**
## Description

Make the command appear and do not hide it.

## Related Issue(s)

Fixes IDE-190

## How to test

```
gp --help
```

#### Preview status

gitpod:summary

## Build Options

<details>
<summary>Build</summary>

- [ ] /werft with-werft
      Run the build with werft instead of GHA
- [ ] leeway-no-cache
- [ ] /werft no-test
      Run Leeway with `--dont-test`
</details>

<details>
<summary>Publish</summary>

- [ ] /werft publish-to-npm
- [ ] /werft publish-to-jb-marketplace
</details>

<details>
<summary>Installer</summary>

- [ ] analytics=segment
- [ ] with-dedicated-emulation
- [ ] workspace-feature-flags
  Add desired feature flags to the end of the line above, space separated
</details>

<details>
<summary>Preview Environment</summary>

- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [ ] /werft with-preview
- [ ] /werft with-large-vm
- [ ] /werft with-gce-vm
      If enabled this will create the environment on GCE infra
- [ ] with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`
</details>

/hold

### CODE REVIEW

**Review by akosyakov — APPROVED** [review]

**Comment by filiptronicek** [comment]
I'm pushing changes that update our dependencies to resolve this:
```log
gitpod /workspace/gitpod/components/gitpod-cli (ide/gp-idp-visible) $ go run .
# github.com/gitpod-io/gitpod/common-go/log
../common-go/log/log.go:150:17: undefined: scrubber.TrustedValue
```

cc @akosyakov

**Comment by akosyakov** [comment]
@filiptronicek sure, but why does it happen now? we are in the same repo

**Comment by filiptronicek** [comment]
It's a good question, I also expected it to not behave like this and always have the latest version...

**Comment by akosyakov** [comment]
there is no [such record](https://github.com/gitpod-io/gitpod/blob/b766c9146ebd90da00547eeae718d371c533fc2c/components/gitpod-cli/go.mod#L67) for scrubber

try to run leeway link from gp-cli

**Comment by filiptronicek** [comment]
leeway link seemed to not do anything

File: components/gitpod-cli/go.mod [hunk 1]
```diff
@@ -19,19 +19,19 @@ require (
...
-	google.golang.org/grpc v1.52.3
+	google.golang.org/grpc v1.56.1
 	gopkg.in/yaml.v2 v2.4.0
 )
 
 require (
-	github.com/gitpod-io/gitpod/components/scrubber v0.0.0-20230530092704-f7d81a8eef6f // indirect
-	github.com/grpc-ecosystem/go-grpc-middleware v1.3.0 // indirect
+	github.com/gitpod-io/gitpod/components/scrubber v0.0.0-20230630070048-0c6b015422ee // indirect
+	github.com/grpc-ecosystem/go-grpc-middleware v1.4.0 // indirect
```
Comment (filiptronicek) [inline components/gitpod-cli/go.mod:34]: seems it's updated here

**Comment by akosyakov** [comment]
It does for me, do you specify a component?

i.e. something like `leeway link .:app`

Screenshot 2023-06-30 at 10 51 43

**Comment by filiptronicek** [comment]
That was it, thanks!

**Pull request merged**